packages/presentation/src/loader/LiveQueries.tsx

```diff
@@ -480,6 +480,10 @@ export function turboChargeResultIfSourceMap<T = unknown>(
       }
       return changedValue
     },
-    perspective,
+    // TODO: Update applySourceDocuments to support releases.
```

Comment:
I'm not sure what this `applySourceDocuments` really do and how we could update it. Added  a follow up ticket in our project so we don't miss it. Would be amazing if you could add more context to that one. See **COREL-286**

Reply:
It lives in `@sanity/client`, after I cut a release I'll bring some context on linear :+1: